In [1]:
import requests 
import pandas as pd



url="https://api.themoviedb.org/3/trending/movie/day?[My_API_KEY]"
response=requests.get(url)
data=response.json()
rows=data["results"]
df=pd.DataFrame(rows)

In [81]:
df

adult                     backdrop_path                   genre_ids  \
0   False  /928wR5R0vmbLsqxJaWmaMHS7SpO.jpg                [80, 18, 36]   
1   False  /nGJpQCAn2NKeDoEflLI5DIvsqoQ.jpg                [35, 18, 53]   
2   False  /mBBJ3N3an8FLkp0ZpGgIJwHKhBP.jpg                [35, 80, 18]   
3   False  /tLCDKsXo6D84IVFanoElosSEKdp.jpg         [12, 16, 35, 10751]   
4   False  /xJWPZIYOEFIjZpBL7SVBGnzRYXp.jpg  [12, 16, 35, 10402, 10751]   
5   False  /j8uPZSPjKYzHlN2B4NhPQVWWZ5x.jpg          [35, 27, 9648, 53]   
6   False  /8moTOzunF7p40oR5XhlDvJckOSW.jpg                    [18, 27]   
7   False  /1NSjj7AnAbAUXHAJv3a14yC3h4E.jpg             [35, 18, 10749]   
8   False  /k2WyDw2NTUIWnuEs5gT7wgrCQg6.jpg                        [28]   
9   False  /mFcfdnAYaQQhXhKcoITiDYtuO5b.jpg                        [18]   
10  False  /wgNYjRmicuFFnzXeXH9KszVIJoZ.jpg               [18, 878, 53]   
11  False  /n6bUvigpRFqSwmPp1m2YADdbRBc.jpg                [80, 18, 53]   
12  False  /qAhedRxRYWZAgZ8O8pHIl6QHdD7.jpg                [28, 12, 35]   
13  False  /nRXO2SnOA75OsWhNhXstHB8ZmI3.jpg                    [12, 18]   
14  False  /yRNrOJyyeHJjgtJNLcqpo7rQyS6.jpg                    [18, 36]   
15  False  /AbRYlvwAKHs0YuyNO6NX9ofq4l6.jpg              [35, 9648, 53]   
16  False  /spYx9XQFODuqEVoPpvaJI1ksAVt.jpg                    [28, 53]   
17  False  /7RyHsO4yDXtBv1zUU3mTpHeQ0d5.jpg               [28, 12, 878]   
18  False  /uF4gCZHiui3UJ2BzUb8Nr6De6Lt.jpg                        [18]   
19  False  /5myQbDzw3l8K9yofUXRJ4UTVgam.jpg               [28, 12, 878]   

        id media_type original_language  \
0   398978      movie                en   
1   466272      movie                en   
2   540901      movie                en   
3   431580      movie                en   
4   330457      movie                en   
5   567609      movie                en   
6   474350      movie                en   
7   639609      movie                en   
8   423204      movie                en   
9   535544      movie                en   
10  539892      movie                en   
11  475557      movie                en   
12  384018      movie                en   
13  420818      movie                en   
14  524348      movie                en   
15  546554      movie                en   
16  522938      movie                en   
17  299534      movie                en   
18  472674      movie                en   
19  429617      movie                en   

                           original_title  \
0                            The Irishman   
1        Once Upon a Time... in Hollywood   
2                                Hustlers   
3                              Abominable   
4                               Frozen II   
5                            Ready or Not   
6                          It Chapter Two   
7                            Holiday Rush   
8                        Angel Has Fallen   
9                           Downton Abbey   
10                                 Freaks   
11                                  Joker   
12  Fast & Furious Presents: Hobbs & Shaw   
13                          The Lion King   
14                             The Report   
15                             Knives Out   
16                      Rambo: Last Blood   
17                      Avengers: Endgame   
18                          The Goldfinch   
19              Spider-Man: Far from Home   

                                             overview  popularity  \
0   Pennsylvania, 1956. Frank Sheeran, a war veter...     113.763   
1   A faded television actor and his stunt double ...     259.074   
2   A crew of savvy former strip club employees ba...     156.277   
3   A group of misfits encounter a young Yeti name...      36.932   
4   Elsa, Anna, Kristoff and Olaf head far into th...     363.690   
5   A bride's wedding night takes a sinister turn ...      97.140   
6   27 years after overcoming the malevolent super...      96.111   
7   After his sudden firin

In [82]:
df[['title','id']]

title      id
0                            The Irishman  398978
1        Once Upon a Time... in Hollywood  466272
2                                Hustlers  540901
3                              Abominable  431580
4                               Frozen II  330457
5                            Ready or Not  567609
6                          It Chapter Two  474350
7                            Holiday Rush  639609
8                        Angel Has Fallen  423204
9                           Downton Abbey  535544
10                                 Freaks  539892
11                                  Joker  475557
12  Fast & Furious Presents: Hobbs & Shaw  384018
13                          The Lion King  420818
14                             The Report  524348
15                             Knives Out  546554
16                      Rambo: Last Blood  522938
17                      Avengers: Endgame  299534
18                          The Goldfinch  472674
19              Spider-Man: Far from Home  429617

## Get Movie Review for movie id 550

In [2]:
url="https://api.themoviedb.org/3/movie/550/reviews?[My_API_KEY]"
response=requests.get(url)
data=response.json()
rows=data["results"]
df=pd.DataFrame(rows)

In [3]:
df

author                                            content  \
0       Goddard  Pretty awesome movie.  It shows what one crazy...   
1  Brett Pascoe  In my top 5 of all time favourite movies. Grea...   

                         id                                                url  
0  5b1c13b9c3a36848f2026384  https://www.themoviedb.org/review/5b1c13b9c3a3...  
1  5b3e1ba1925141144c007f17  https://www.themoviedb.org/review/5b3e1ba19251...

## Get Movie Review for movie id 420818

In [4]:
url="https://api.themoviedb.org/3/movie/420818/reviews?[MY_API_KEY]"
response=requests.get(url)
data=response.json()
rows=data["results"]
df=pd.DataFrame(rows)

In [5]:
df

author                                            content  \
0     SWITCH.  ‘The Lion King’ is a catastrophe; a new low in...   
1  msbreviews  If you enjoy reading my Spoiler-Free reviews, ...   

                         id                                                url  
0  5d2ddc7d6a300b0011a469df  https://www.themoviedb.org/review/5d2ddc7d6a30...  
1  5d31d3991d78f2000ee0b492  https://www.themoviedb.org/review/5d31d3991d78...

In [6]:
review_list=df['content']

In [91]:
review_list[0]

"‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes. There’s nothing redeeming about it, with every decision either ill-conceived or mishandled to the point of incompetence. In Favreau’s hands, ‘The Lion King’ is rendered thunderously dull, lacking in any tension or complex characterisation, taking a laboriously long time to go nowhere and never once justifying its contentious existence. Even with my dislike of the original, I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film. If nothing else, this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy. They don’t care whether the film is good or whether you enjoy it. All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can, and maybe if they throw some recognisable iconic moments from your c

## Get Movie Review for movie id 466272

In [76]:
url="https://api.themoviedb.org/3/movie/466272/reviews?[MY_API_KEY]"
response=requests.get(url)
data=response.json()
rows=data["results"]
df=pd.DataFrame(rows)

In [77]:
df

author                                            content  \
0     SWITCH.  The movie isn’t for everyone, of course, but i...   
1  msbreviews  If you enjoy reading my Spoiler-Free reviews, ...   
2       Gimly  I'm not here to explain _Once Upon a Time In H...   
3      Wuchak  ***Tarantino’s revenge on the Manson psychos**...   

                         id                                                url  
0  5d4a57a8439be10015910b94  https://www.themoviedb.org/review/5d4a57a8439b...  
1  5d5e9465f263ba752c944e09  https://www.themoviedb.org/review/5d5e9465f263...  
2  5d625e316dea3a746b9457e8  https://www.themoviedb.org/review/5d625e316dea...  
3  5d7ee2a83c43440998a16f94  https://www.themoviedb.org/review/5d7ee2a83c43...

In [78]:
review_list=df['content']

In [79]:
review_list[3]

'***Tarantino’s revenge on the Manson psychos***\r\n\r\nIn the late 60s, Rick Dalton (Leonardo DiCaprio), a popular TV Western actor, finds his career taking a downturn and tries to recover with the encouragement of his kick-axx stunt double and best friend, Cliff Booth (Brad Pitt). Sharon Tate (Margo Robbie) & Roman Polanski are neighbors with Jay Sebring always hanging around (Emile Hirsch). Meanwhile the Manson Family nutjobs are lurking in the background, prepping to attack. \r\n\r\n“Once Upon a Time… in Hollywood” (2019) is Quentin Tarantino’s 9th full film and, for me, ranks somewhere in the middle of his oeuvre. It may not be as great as “Pulp Fiction” (1994) and “Django Unchained” (2012), but it places well with “Inglourious Basterds” (2009), “Jackie Brown” (1997) and “The Hateful Eight” (2015).  \r\n\r\nA famous director once succinctly defined a great movie as such: Three good scenes, no bad scenes. While the second part of this definition is debatable with "Once Upon a Time.

In [44]:
review_list[1]

'If you enjoy reading my Spoiler-Free reviews, please follow my blog :)\r\n\r\nQuentin Tarantino is one of the best filmmakers of all-time. He has undeniable talent behind the camera, and his movies are fated to leave a mark in each year they’re released. In addition to that, he’s also an extraordinary screenwriter, as Once Upon a Time in Hollywood proves once again. His knowledge of the early decades of film is vast, so every feature he produces is always going to be filled with references to those “fairy tale” years. And that’s precisely what this movie is: a fairy tale in Hollywood, hence its title. Let me just leave this here right off the bat: I’m not going to address any controversy surrounding this film (namely, the whole Bruce Lee depiction and the Manson Family, in general), as I’m always fair and impartial to the movie I’m reviewing. Moving on …\r\n\r\nMy knowledge of the 60s isn’t that good. Obviously, I know the whole Sharon Tate story, as well as the famous Manson murders,

In [45]:
review_list[2]

"I'm not here to explain _Once Upon a Time In Hollywood_, just to enjoy it.\r\n\r\n_Final rating:★★★★ - Very strong appeal. A personal favourite._"

## Get Movie review for movie id 567609

In [72]:
url="https://api.themoviedb.org/3/movie/567609/reviews?[My_API_KEY]"
response=requests.get(url)
data=response.json()
rows=data["results"]
df=pd.DataFrame(rows)

In [73]:
df

author                                            content  \
0        msbreviews  If you enjoy reading my Spoiler-Free reviews, ...   
1  Stephen Campbell  **_An entertaining horror-comedy that takes ai...   
2           SWITCH.  ‘Ready or Not’ doesn’t sport as many laughs as...   

                         id                                                url  
0  5d5d7b1ba3d02700160bfc08  https://www.themoviedb.org/review/5d5d7b1ba3d0...  
1  5da13750b84f94001597c5d4  https://www.themoviedb.org/review/5da13750b84f...  
2  5db02cac09dda40011e0dd46  https://www.themoviedb.org/review/5db02cac09dd...

In [74]:
review_list=df['content']

In [49]:
review_list

0    If you enjoy reading my Spoiler-Free reviews, ...
1    **_An entertaining horror-comedy that takes ai...
2    ‘Ready or Not’ doesn’t sport as many laughs as...
Name: content, dtype: object

In [93]:
review_list[0]

"‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes. There’s nothing redeeming about it, with every decision either ill-conceived or mishandled to the point of incompetence. In Favreau’s hands, ‘The Lion King’ is rendered thunderously dull, lacking in any tension or complex characterisation, taking a laboriously long time to go nowhere and never once justifying its contentious existence. Even with my dislike of the original, I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film. If nothing else, this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy. They don’t care whether the film is good or whether you enjoy it. All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can, and maybe if they throw some recognisable iconic moments from your c

In [51]:
review_list[1]

'**_An entertaining horror-comedy that takes aim at the decadence and insularity of the 1%_**\r\n\r\n>_So distribution should undo excess,/And each man have enough._\r\n\r\n- William Shakespeare; _His True Chronicle Historie of the life and death of King Lear, and his three Daughters. With the vnfortunate life of Edgar, sonne and heire to the Earle of Gloster, and his sullen and assumed humour of Tom of Bedlam_, 4.i.73-74 (1605-1606)\r\n\r\n>_In the long run, men inevitably become the victims of their wealth. They adapt their lives and habits to their money, not their money to their lives. It preoccupies t__heir thoughts, creates artificial needs, and draws a curtain between them and the world._\r\n\r\n- Herbert Croly; _The Promise of American Life_ (1909)\r\n\r\n>_I don\'t want to be married just to be married. I can\'t think of anything lonelier than spending the rest of my life with someone I can\'t talk to, or worse, someone I can\'t be silent with._\r\n\r\n- Mary Ann Shaffer; _The

In [52]:
review_list[2]

"‘Ready or Not’ doesn’t sport as many laughs as you'd want for this ridiculous story, nor is it dark enough to be a true horror; it's just gory and violent with the occasional jump scare and anxiety-ridden scene. So while it doesn’t meet expectations or desires, it’s still a fun, thrilling, wicked romp through the blood-soaked mansion and goat barn. \r\n\r\nSo prepare yourself for squeals of fear, squeals of delight, squeals of patriotism and the occasional giggled “WTF!?”... because here it comes!\r\n- Jess Fenton\r\n\r\nRead Jess' full article...\r\nhttps://www.maketheswitch.com.au/article/review-ready-or-not-not-quite-horror-not-quite-comedy-but-a-wickedly-good-time"

## Saving DataFrame to CSV file

In [53]:
df.to_csv('moviesdb_review1.csv')

# Start with one review[0]

In [92]:
text=review_list[0]

In [94]:
text

"‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes. There’s nothing redeeming about it, with every decision either ill-conceived or mishandled to the point of incompetence. In Favreau’s hands, ‘The Lion King’ is rendered thunderously dull, lacking in any tension or complex characterisation, taking a laboriously long time to go nowhere and never once justifying its contentious existence. Even with my dislike of the original, I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film. If nothing else, this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy. They don’t care whether the film is good or whether you enjoy it. All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can, and maybe if they throw some recognisable iconic moments from your c

In [109]:
from nltk.stem import PorterStemmer, WordNetLemmatizer
from nltk.corpus import wordnet

In [110]:
import nltk
nltk.download('punkt')

[nltk_data] Downloading package punkt to C:\Users\Su
[nltk_data]     Gnee\AppData\Roaming\nltk_data...
[nltk_data]   Package punkt is already up-to-date!


True

In [97]:
sentences=nltk.sent_tokenize(text)

In [98]:
for sentence in sentences:
    print(sentence)
    print()

‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes.

There’s nothing redeeming about it, with every decision either ill-conceived or mishandled to the point of incompetence.

In Favreau’s hands, ‘The Lion King’ is rendered thunderously dull, lacking in any tension or complex characterisation, taking a laboriously long time to go nowhere and never once justifying its contentious existence.

Even with my dislike of the original, I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film.

If nothing else, this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy.

They don’t care whether the film is good or whether you enjoy it.

All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can, and maybe if they throw some recognisable iconic moments from y

In [99]:
for sentence in sentences:
    words=nltk.word_tokenize(sentence)
    print(words)
    print()

['‘', 'The', 'Lion', 'King', '’', 'is', 'a', 'catastrophe', ';', 'a', 'new', 'low', 'in', 'the', 'ever-diminishing', 'returns', 'of', 'Disney', '’', 's', 'endless', 'run', 'of', 'remakes', '.']

['There', '’', 's', 'nothing', 'redeeming', 'about', 'it', ',', 'with', 'every', 'decision', 'either', 'ill-conceived', 'or', 'mishandled', 'to', 'the', 'point', 'of', 'incompetence', '.']

['In', 'Favreau', '’', 's', 'hands', ',', '‘', 'The', 'Lion', 'King', '’', 'is', 'rendered', 'thunderously', 'dull', ',', 'lacking', 'in', 'any', 'tension', 'or', 'complex', 'characterisation', ',', 'taking', 'a', 'laboriously', 'long', 'time', 'to', 'go', 'nowhere', 'and', 'never', 'once', 'justifying', 'its', 'contentious', 'existence', '.']

['Even', 'with', 'my', 'dislike', 'of', 'the', 'original', ',', 'I', 'was', 'flabbergasted', 'at', 'how', 'thoroughly', 'this', 'film', 'never', 'attempts', 'to', 'understand', 'why', 'so', 'many', 'people', 'love', 'the', '1994', 'film', '.']

['If', 'nothing', 'else

## Get the first sentence from the array

In [113]:
print(sentences[0])

‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes.


## Data Cleaning - Remove Punctuation
# Replace non words with space

In [114]:
import re

In [115]:
print(sentences[0])

‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes.


In [116]:
pattern=r"[^\w]"

In [117]:
sentence_without_punctuation=re.sub(pattern," ",sentences[0])

In [118]:
print(sentence_without_punctuation)

 The Lion King  is a catastrophe  a new low in the ever diminishing returns of Disney s endless run of remakes 


In [119]:
words=nltk.word_tokenize(sentence_without_punctuation)
print(words)

['The', 'Lion', 'King', 'is', 'a', 'catastrophe', 'a', 'new', 'low', 'in', 'the', 'ever', 'diminishing', 'returns', 'of', 'Disney', 's', 'endless', 'run', 'of', 'remakes']


In [120]:
from nltk.corpus import stopwords


In [121]:
print(stopwords.words("english"))

['i', 'me', 'my', 'myself', 'we', 'our', 'ours', 'ourselves', 'you', "you're", "you've", "you'll", "you'd", 'your', 'yours', 'yourself', 'yourselves', 'he', 'him', 'his', 'himself', 'she', "she's", 'her', 'hers', 'herself', 'it', "it's", 'its', 'itself', 'they', 'them', 'their', 'theirs', 'themselves', 'what', 'which', 'who', 'whom', 'this', 'that', "that'll", 'these', 'those', 'am', 'is', 'are', 'was', 'were', 'be', 'been', 'being', 'have', 'has', 'had', 'having', 'do', 'does', 'did', 'doing', 'a', 'an', 'the', 'and', 'but', 'if', 'or', 'because', 'as', 'until', 'while', 'of', 'at', 'by', 'for', 'with', 'about', 'against', 'between', 'into', 'through', 'during', 'before', 'after', 'above', 'below', 'to', 'from', 'up', 'down', 'in', 'out', 'on', 'off', 'over', 'under', 'again', 'further', 'then', 'once', 'here', 'there', 'when', 'where', 'why', 'how', 'all', 'any', 'both', 'each', 'few', 'more', 'most', 'other', 'some', 'such', 'no', 'nor', 'not', 'only', 'own', 'same', 'so', 'than', '

In [122]:
stop_words=set(stopwords.words("english"))

In [123]:
without_stop_words=[word for word in words if not word in stop_words]

In [124]:
print(without_stop_words)

['The', 'Lion', 'King', 'catastrophe', 'new', 'low', 'ever', 'diminishing', 'returns', 'Disney', 'endless', 'run', 'remakes']


## Lemmatization

In [125]:
stemmer=PorterStemmer()

In [126]:
stemmed_overview=[stemmer.stem(word) for word in without_stop_words]

In [127]:
print(stemmed_overview)

['the', 'lion', 'king', 'catastroph', 'new', 'low', 'ever', 'diminish', 'return', 'disney', 'endless', 'run', 'remak']


In [128]:
from nltk.tokenize.treebank import TreebankWordDetokenizer

In [129]:
detokenized_string=TreebankWordDetokenizer().detokenize(stemmed_overview)

# Sentiment Analysis

In [130]:
from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer

In [131]:
analyser= SentimentIntensityAnalyzer()

In [132]:
score=analyser.polarity_scores(detokenized_string)


In [133]:
print(score)

{'neg': 0.149, 'neu': 0.851, 'pos': 0.0, 'compound': -0.2732}


Our sentence was graded 14.9% positive, 85.1% neutral 0% positive. All these add up to 1

# New Process to do a Sentiment Analysis for Lion King review

In [7]:
url="https://api.themoviedb.org/3/movie/420818/reviews?[MY_API_KEY]"
response=requests.get(url)
data=response.json()
rows=data["results"]
df=pd.DataFrame(rows)

In [8]:
df

author                                            content  \
0     SWITCH.  ‘The Lion King’ is a catastrophe; a new low in...   
1  msbreviews  If you enjoy reading my Spoiler-Free reviews, ...   

                         id                                                url  
0  5d2ddc7d6a300b0011a469df  https://www.themoviedb.org/review/5d2ddc7d6a30...  
1  5d31d3991d78f2000ee0b492  https://www.themoviedb.org/review/5d31d3991d78...

In [9]:
review_list=df['content']

In [10]:
review_list[0]

"‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes. There’s nothing redeeming about it, with every decision either ill-conceived or mishandled to the point of incompetence. In Favreau’s hands, ‘The Lion King’ is rendered thunderously dull, lacking in any tension or complex characterisation, taking a laboriously long time to go nowhere and never once justifying its contentious existence. Even with my dislike of the original, I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film. If nothing else, this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy. They don’t care whether the film is good or whether you enjoy it. All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can, and maybe if they throw some recognisable iconic moments from your c

In [11]:
my_sentences=review_list[0]

In [12]:
print(my_sentences)

‘The Lion King’ is a catastrophe; a new low in the ever-diminishing returns of Disney’s endless run of remakes. There’s nothing redeeming about it, with every decision either ill-conceived or mishandled to the point of incompetence. In Favreau’s hands, ‘The Lion King’ is rendered thunderously dull, lacking in any tension or complex characterisation, taking a laboriously long time to go nowhere and never once justifying its contentious existence. Even with my dislike of the original, I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film. If nothing else, this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy. They don’t care whether the film is good or whether you enjoy it. All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can, and maybe if they throw some recognisable iconic moments from your ch

In [15]:
# Write a pattern to match all words

pattern=r"[^\w]"

In [17]:
# substitute the pattern found with a space
import re
sentences_without_punctuation=re.sub(pattern," ",my_sentences)

In [18]:
print(sentences_without_punctuation)

 The Lion King  is a catastrophe  a new low in the ever diminishing returns of Disney s endless run of remakes  There s nothing redeeming about it  with every decision either ill conceived or mishandled to the point of incompetence  In Favreau s hands   The Lion King  is rendered thunderously dull  lacking in any tension or complex characterisation  taking a laboriously long time to go nowhere and never once justifying its contentious existence  Even with my dislike of the original  I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the 1994 film  If nothing else  this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy  They don t care whether the film is good or whether you enjoy it  All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can  and maybe if they throw some recognisable iconic moments from your ch

In [20]:
# find all letters
# substitute the pattern found with a space

pattern=r"[0-9]\w*"

In [21]:
#print the result of finding this numerals in the sentence_withou_punctuation
print(re.findall(pattern, sentences_without_punctuation))

['1994']


In [22]:
# substitute pattern found with space and print the result
sentences_only_alphabets=re.sub(pattern," ",sentences_without_punctuation)

In [23]:
print(sentences_only_alphabets)

 The Lion King  is a catastrophe  a new low in the ever diminishing returns of Disney s endless run of remakes  There s nothing redeeming about it  with every decision either ill conceived or mishandled to the point of incompetence  In Favreau s hands   The Lion King  is rendered thunderously dull  lacking in any tension or complex characterisation  taking a laboriously long time to go nowhere and never once justifying its contentious existence  Even with my dislike of the original  I was flabbergasted at how thoroughly this film never attempts to understand why so many people love the   film  If nothing else  this film makes it abundantly clear that Disney has no interest in making great cinema or honouring its own legacy  They don t care whether the film is good or whether you enjoy it  All they care about is using nostalgia to trick you into buying your ticket so they can make as much money off you as they can  and maybe if they throw some recognisable iconic moments from your child

In [29]:
from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer

In [30]:
analyser= SentimentIntensityAnalyzer()

In [31]:
score=analyser.polarity_scores("The Lion King is a bad")


In [32]:
print(score)

{'neg': 0.467, 'neu': 0.533, 'pos': 0.0, 'compound': -0.5423}


Sentiment analysis shows that 46.7% is negative, 53.3% is neutral 0% is positive